добавила проект

vitominx opened the pull request.

vitominx committed:
добавила проект

maoleshko commented on appModules/controllers/game.js:
```diff
@@ -0,0 +1,18 @@
+const { config } = require("../rating");
+const { getRandomGame } = require("../api");
+
+async function gameRouteController(res) {
```
Для отображения игры на главной странице используем асинхронную функцию gameRouteController, которая обрабатывает запрос и возвращает случайную игру из файла 'rating.json'.
async function gameRouteController(res) {
  fs.readFile("./dataset/rating.json", (err, ratingFile) => {
    if (err) {
      res.statusCode = 500;
      res.end(`Internal Server Error`);
    }
    const data = JSON.parse(ratingFile);
    const game = getRandomGame(data);
    res.setHeader("Content-Type", "application/json");
    res.end(JSON.stringify(game));
  });
}

maoleshko commented on appModules/rating/config.js:
```diff
@@ -0,0 +1,11 @@
+const PATH_TO_RATING_FILE = require("../dataset/rating.json");
```
путь к файлу без require, просто 
const PATH_TO_RATING_FILE = "./dataset/rating.json";

maoleshko commented on appModules/api/config.js:
```diff
@@ -0,0 +1,6 @@
+const BASE_URL = "https://api-code-2.practicum-team.ru/games";
```
базовый url без /games на конце
const BASE_URL = "https://api-code-2.practicum-team.ru";

vitominx committed:
исправила ошибки

сайт теперь открывается, но видно, что имеются проблемы со стилями, но так и не поняла, как именно их исправить
